moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -47,11 +48,20 @@
 #include <moveit/profiler/profiler.h>
 #include <moveit/utils/lexical_casts.h>
 
+#include <ompl/config.h>
 #include <ompl/base/samplers/UniformValidStateSampler.h>
 #include <ompl/base/goals/GoalLazySamples.h>
 #include <ompl/tools/config/SelfConfig.h>
 #include <ompl/base/spaces/SE3StateSpace.h>
 #include <ompl/datastructures/PDF.h>
+#if OMPL_VERSION_VALUE < 1005000
```

Comment:
Add TODO to remove this if/else after support for ROS Kinetic(?) is dropped? This feature exists in Melodic, right?

Reply:
The feature only exists in OMPL master, which will be released as OMPL 1.5.0 in the near future. Current Melodic works fine with building this version of OMPL from source. The binary releases will target Noetic, since the ABI has changed.